api/cmd/api/setup.go

```diff
@@ -56,6 +56,16 @@ func initDB(cfg config.DatabaseConfig) (*gorm.DB, func()) {
 		panic(err)
 	}
 	db.LogMode(false)
+
+	sqlDB := db.DB()
+	if sqlDB == nil {
+		panic("fail to get the underlying database connection")
+	}
+
+	sqlDB.SetConnMaxIdleTime(cfg.ConnMaxIdleTime)
+	sqlDB.SetMaxIdleConns(cfg.MaxIdleConns)
+	sqlDB.SetMaxOpenConns(cfg.MaxOpenConns)
```

Comment:
Can you add `SetConnMaxLifetime` (https://github.com/go-sql-driver/mysql/issues/529) ?

Reply:
Added